README.md

```diff
@@ -1,9 +1,43 @@
-# Alibeez-to-google-calendar-connector
+# Alibeez-to-Google-Calendar Connector
 
-Sync leaves from Alibeez to your Google Agenda account
+Copies leaves from Alibeez to Google Calendar.
```

Comment:
"Sync" was maybe better, as the tool also delete the event in Calendar if the leave is cancelled.

Reply:
I prefer the infinitive form of the verb here, **Sync** or **Copy**.

Reply:
@zigarn Right, I keep switch from one to the other because I'm afraid "sync" might be understood as being both ways (changes in Calendar reflected in Alibeez), but I agree it's still better than copy and also it is used in the code.

Reply:
@Zenigata A matter of preference :)